doc/source/timeseries.rst

```diff
@@ -191,6 +187,25 @@ or ``format``, use ``to_datetime`` if these are required.
 
     pd.Timestamp('2010/11/12')
 
+Providing a Format Argument
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+A specific ``format`` argument can be passed in addition to the datetime string.
```

Comment:
I would add something like "to ensure a specific and consistent parsing of the string"

Reply:
Can you also add a blankline between the title and the first sentence ?